README.md

```diff
@@ -34,48 +34,48 @@ Available Commands:
   help        Help about any command
 
 Flags:
-  -c, --config string     File path to compose file (default "cfn-compose.yml")
```

Comment:
@omegazyadav  let's rename `cfnc.yml` back to `cfn-compose.yml` as we are only changing the name of the cli. Also rename this file `examples/ec2-sg/cfnc.yml`

Reply:
@rbalman I have reverted the corresponding changes. Thanks.